src/ecdh.rs

```diff
@@ -32,13 +32,12 @@ const SHARED_SECRET_SIZE: usize = constants::SECRET_KEY_SIZE;
 /// # Examples
 ///
 /// ```
-/// # #[cfg(all(feature = "std", feature = "rand-std"))] {
-/// # use secp256k1::Secp256k1;
+/// # #[cfg(feature = "rand-std")] {
+/// # use secp256k1::{rand, Secp256k1};
```

Comment:
Agreed. The old formatting was chosen with "this line will be hidden" in mind.

I have a mild preference for indenting vs putting `{` on the next line.

Reply:
Its already laborious to write example code in comments because of zero tooling support, we should be careful what we choose because we (I) will have to go back and change every example in the whole codebase.

Reply:
Ok, then let's do `{` on the next line since that's easiest :P and say that this is a very low priority thing.